source/technical/networks/networks_sdn.rst

```diff
@@ -249,6 +248,56 @@ The TACC site uses a mapping with the UUID of the Chameleon node mapped to the f
    9d05db81-05e5-441b-9462-1e17d86e1a6b     10131
    f59f3140-57a0-4452-98dc-edfbb53f07e1     10132
 
+Corsa DP2000 Virtual Forwarding Contexts: Network Layout and Advanced Features
+______________________________________________________________________________
+
+Virtual Forwarding Contexts (VFC) are the native OpenFlow abstraction used by the Corsa DP2000 series switches. Each VFC can be thought of as a virtual OpenFlow switch.  Chameleon users can create VFCs by creating isolated networks on Chameleon via CLI or using complex appliaces.
+
+In this section, actual rack and switch layout of Skylake Nodes and Corsa DP2000 switches for both Chameleon sites is represented in the following figures. Also, example isolated networks with different controller options are shown along with associated VFCs and tunnels from Skylake Nodes are shown.
+
+Users are able to specify an external OpenFlow controller and can assign a name to their VFCs. If an external controller is not specified, VFC is controlled by the OpenFlow controller (Learning Bridge Application) running on the switch.  
+
+1. Create an isolated network without an external OpenFlow controller and a VFC name:
```

Comment:
This reads like a series of steps one has to perform, but I think they are intended to be isolated examples? Maybe they could be formatted like

```rst
**Example: Create an isolated network without...**

.. code-block:: bash
...
```